src/panels/config/script/ha-script-editor.js

```diff
@@ -232,7 +235,9 @@ class HaScriptEditor extends LocalizeMixin(NavigateMixin(PolymerElement)) {
           this._updateComponent();
         },
         () => {
-          alert("Only scripts inside scripts.yaml are editable.");
+          alert(
+            "[[localize('ui.panel.config.script.editor.load_error_not_editable')]]"
```

Comment:
Remove the ""

Reply:
If I remove the quotes, `npm run lint` throws following errors:

```
  238:17  error  Insert `[`                                                                                                                                                 prettier/prettier
  239:14  error  Replace `[localize('ui.panel.config.script.editor.load_error_not_editable')]]` with `localize("ui.panel.config.script.editor.load_error_not_editable")],`  prettier/prettier
  239:15  error  'localize' is not defined                                                                                                                                  no-undef
  240:11  error  Insert `]`                                                                                                                                                 prettier/prettier
```

After trying to fix the new linting issues, the error `'localize' is not defined` still remains.

Reply:
Yes, because it is not inside the render method of Polymer, use `this.hass.localize`

Reply:
Thank you very much. I think I've got the point now.